expyfun/stimuli/_tracker.py

```diff
@@ -274,13 +287,35 @@ def respond(self, correct):
                     reversals=[int(s) for s in self._reversals],
                     x=[float(s) for s in self._x])))
 
+    def check_valid(self, n_reversals):
+        """See if the last reversals contain x_min/x_max reversals.
```

Comment:
"reversals" at either extreme should be fine if they actually correspond to responses that cause a reverse in direction. So I don't get this wording